boards/arm/nucleo_f429zi/doc/index.rst

```diff
@@ -182,6 +182,22 @@ Programming and Debugging
 The Nucleo F429ZI board includes an ST-LINK/V2-1 embedded debug tool interface.
 This interface is supported by the openocd version included in Zephyr SDK.
 
+Flash partitions for MCUBoot bootloader
+***************************************
+
+The on-board STM32F429ZI MCU has 2MBs of internal flash memory. To use `MCUboot`_,
+define a `Zephyr partition table`_ for the flash memory in its device tree file
```

Comment:
While what you're linking to now will work, it always points to the /latest/ documentation. When we create a tagged version of the documentation, we want references to be to content within that tagged version. So the right way to write this link is:
```
define a :ref:`Zephyr partition table <flash_partitions>` for the flash memory
```
The Sphinx document processor will create a link within the document tree it's building instead of always referencing the /latest/ version.

Reply:
Thanks for pointing out the potential problem, David! I've already corrected it again by following your comments.